docs/configuration_options.md

```diff
@@ -3649,3 +3649,57 @@ Replication factor for Scylla Cloud cluster (default: 3)
 **default:** N/A
 
 **type:** int
+
+
+## **n_vector_nodes** / SCT_N_VECTOR_NODES
+
+Number of vector store nodes (0 = VS is disabled)
+
+**default:** N/A
+
+**type:** int
+
+
+## **vector_store_port** / SCT_VECTOR_STORE_PORT
+
+Vector Store API port
+
+**default:** 6080
+
+**type:** int
+
+
+## **vector_store_scylla_port** / SCT_VECTOR_STORE_SCYLLA_PORT
+
+ScyllaDB connection port for Vector Store
+
+**default:** 9042
+
+**type:** int
+
+
+## **vector_store_threads** / SCT_VECTOR_STORE_THREADS
+
+Vector indexing threads (default: number of CPU cores)
+
+**default:** 2
+
+**type:** int
+
+
+## **vector_store_cpu_limit** / SCT_VECTOR_STORE_CPU_LIMIT
+
+CPU limit for vector store containers (cores)
+
+**default:** 2.0
+
+**type:** float
+
+
+## **vector_store_memory_limit** / SCT_VECTOR_STORE_MEMORY_LIMIT
```

Comment:
sound to be like docker backend specific, and should be marked as such (even in it's name)